install/kubernetes/helm/istio/charts/egressgateway/templates/clusterrole.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.global.rbacEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    app: {{ template "istio.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: istio-egressgateway-{{ .Release.Namespace }}
+rules:
+- apiGroups: ["extensions"]
+  resources: ["thirdpartyresources", "virtualservices", "destinationrules", "gateways"]
+  verbs: ["get", "watch", "list", "update"]
+- apiGroups: [""]
+  resources: ["configmaps", "pods", "endpoints", "services"]
```

Comment:
Curious why egress gateway needs watching  for `"configmaps", "pods", "endpoints", "services"`, does this pod run a controller which needs permissions to Watch for these?

Reply:
Ah, yes, we do not need it, let me remove it.